test/_inc.sh

```diff
@@ -9,7 +9,7 @@ function clean {
 }
 
 function ignite {
-  dart ../../bin/ignite.dart "$@" 3>&-
+  dart --no-sound-null-safety ../../bin/ignite.dart "$@" 3>&-
```

Comment:
sadly one of our deps, prompts, is still not migrated, but there is a PR up for it
https://github.com/thosakwe/prompts